mdpow/templates/runinput.yml

```diff
@@ -30,9 +32,9 @@ setup:
     boxtype: dodecahedron
          # box type understood by Gromacs 'gmx editconf' (cubic,
          # triclinic, octahedron, dodecahedron
-    gromacsoutput: False
+    gromacsoutput: True
```

Comment:
The default `True` leads to much more output by default but makes debugging a lot easier. Set to `False` for production-ready code.